support/updateStylelintCustomSassFunctions.ts

```diff
@@ -0,0 +1,65 @@
+/**
+ * This script updates a variable from the stylelint config file with a list of custom Sass functions found in the project.
+ * This helps stylelint flag unknown functions that may be unintentionally used.
+ */
+
+import * as fs from 'fs';
+import * as path from 'path';
+
+console.info('Scanning custom functions for Stylelint config update.');
+
+const rootDirectory = path.join(__dirname, '..');
+
+function collectSassFiles(dir: string): string[] {
+  const sassFiles: string[] = [];
+
+  try {
+    fs.readdirSync(dir, { withFileTypes: true }).forEach(dirent => {
+      const fullPath = path.join(dir, dirent.name);
+
+      if (dirent.isDirectory() && dirent.name !== 'node_modules') {
+        sassFiles.push(...collectSassFiles(fullPath)); // Recursively collect files
```

Comment:
readdir has a `recursive` option starting in Node v20

https://nodejs.org/docs/latest-v20.x/api/fs.html#fsreaddirsyncpath-options

Reply:
Noice!